content/en/admin/tootctl.md

```diff
@@ -649,6 +649,15 @@ Remove locally cached copies of media attachments from other servers.
 `--concurrency N`
 : The number of workers to use for this task. Defaults to N=5.
 
+`--prune_profiles`
```

Comment:
I believe dash is preferred to underscore, like `--prune-profiles`

Reply:
But your file changes show underscore? Which one is it?

https://github.com/mastodon/mastodon/pull/22149/files#diff-b5b0d1320d8f699c2587e011ab8c40e986d7a977b8cc6dc6fee1e1db0f2d6d1fR17

Reply:
Yes, please use dashes, the discrepancy is just down to how the `Thor` library works and accommodates the Ruby syntax.

Reply:
@ClearlyClaire Ah, excellent! I didn't know that.

I've amended the commit!